### PR TITLE
[generator] Fixed the file cleanup in altitudes_test.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

File: platform/platform_tests_support/scoped_file.cpp [hunk 1]
```diff
@@ -22,16 +24,35 @@ ScopedFile::ScopedFile(string const & relativePath)
     : m_fullPath(my::JoinFoldersToPath(GetPlatform().WritableDir(), relativePath)),
       m_reset(false)
 {
+  try
```
Comment (ygorshenin) [inline platform/platform_tests_support/scoped_file.cpp]: This block of code looks suspicious - are you sure that we need to do that?

Comment (ygorshenin) [reply platform/platform_tests_support/scoped_file.cpp]: Please, call the second ctor with empty contents.

Comment (mpimenov) [reply platform/platform_tests_support/scoped_file.cpp]: I've changed the constructors.

File: platform/platform_tests_support/scoped_file.cpp [hunk 2]
```diff
@@ -18,20 +20,22 @@ namespace platform
 {
 namespace tests_support
 {
-ScopedFile::ScopedFile(string const & relativePath)
-    : m_fullPath(my::JoinFoldersToPath(GetPlatform().WritableDir(), relativePath)),
-      m_reset(false)
-{
-}
+ScopedFile::ScopedFile(string const & relativePath) : ScopedFile(relativePath, "" /* contents */) {}
```
[the remark this answers is not in the document]
Comment (mpimenov) [reply platform/platform_tests_support/scoped_file.cpp]: done

**Pull request merged**